lib/tasks/one_off.rake

```diff
@@ -1,4 +1,20 @@
 namespace :pvb do
+  desc 'Merge Isle Of Wight Prisons'
+  task merge_iow: :environment do
+    albany    = Estate.find_by!(nomis_id: 'ALI').prisons.first
+    parkhurst = Estate.find_by!(nomis_id: 'IWI').prisons.first
+    FeedbackSubmission.where(prison_id: albany.id).
+      find_in_batches do |feebback_submissions|
+
+      FeedbackSubmission.where(id: feebback_submissions.map(&:id)).
+        update_all(prison_id: parkhurst.id)
+    end
+
+    Visit.where(prison_id: albany.id).find_in_batches do |visits|
+      Visit.where(id: visits.map(&:id)).update_all(prison_id: parkhurst.id)
+    end
+  end
```

Comment:
Be careful when you modify the set you are iterating over, what can happen is that only half the records get updated. I'm not sure if this will happen with this query.

Easy to run multiple times if necessary. The alternative is to avoid modifying the set you are looping by doing an update all with a limit and keep looking until there are no more records to update.

Something like:

```ruby
while FeedbackSubmission.where(prison_id: albany.id).any? do
  FeedbackSubmission.
    where(prison_id: albany.id).
    limit(1000).
    update_all(prison_id: parkhurst.id)
end
```